src/essreflectometry/amor/load.py

```diff
@@ -64,7 +64,7 @@ def _assemble_event_data(dg: sc.DataGroup) -> sc.DataArray:
     :
         A data array with the events extracted from ``dg``.
     """
-    events = dg['instrument']['multiblade_detector']['data']
+    events = dg['instrument']['multiblade_detector']['data'].copy(deep=False)
     events.bins.coords['tof'] = events.bins.coords.pop('event_time_offset')
```

Comment:
If the `copy` is meant to prevent in-place modification, not that is does *not* affect the bin content, i.e., the line second line modifies the buffer of the original. See scipp/scipp#2773.